x-pack/plugins/infra/public/pages/metrics/hosts/hooks/use_hosts_table.test.ts

```diff
@@ -10,9 +10,13 @@ import { renderHook } from '@testing-library/react-hooks';
 import { InfraAssetMetricsItem } from '../../../../../common/http_api';
 import * as useUnifiedSearchHooks from './use_unified_search';
 import * as useHostsViewHooks from './use_hosts_view';
+import * as useKibanaContextForPluginHook from '../../../../hooks/use_kibana';
+import * as useMetricsDataViewHooks from './use_data_view';
 
 jest.mock('./use_unified_search');
 jest.mock('./use_hosts_view');
+jest.mock('./use_data_view');
```

Comment:
In the test I needed to add some mocks to the hooks I am using, nothing else changes here